charts/icantbelieveitsnotvaletudo/templates/NOTES.txt

```diff
@@ -0,0 +1,6 @@
+{{- if .Values.ingress.enabled }}
```

Comment:
The `NOTES.txt` only outputs something if an ingress is configured.  Have you considered adding extra information for cases when there is no ingress?  Take [jackett's `NOTES.txt`](https://github.com/billimek/billimek-charts/blob/master/charts/jackett/templates/NOTES.txt) as an example?

Reply:
My reason for it being that this app doesn't take any http input at all if the webserver is disabled - it will work purely through the provided mqtt server (and will crash if it can't connect). And now while I type this i see your point :D Will having no output when the webserver is disabled be ok?